task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -210,7 +210,7 @@ def __update_modified_pod(self, pod: V1Pod, event: Optional[PodEvent]) -> None:
 
         if pod.status.phase not in SUPPORTED_POD_MODIFIED_EVENT_PHASES:
             logger.debug(
-                f"Got a MODIFIED event for {pod_name} for unhandled phase: "
+                f"Got a {event['type']} event for {pod_name} for unhandled phase: "
```

Comment:
i wonder if we can do something like:
```python
    def __update_modified_pod(self, pod: V1Pod, event: Optional[PodEvent]) -> None:
        ...
        event_type = event['type'] if event else "null"
        ...
```
to handle the null event case (from reconciling)